docs/workflows.rst

```diff
@@ -1,36 +1,47 @@
 .. include:: links.rst
 
-=========
-Workflows
-=========
-
-Basic workflow (no fieldmaps)
-=============================
-
-``fmriprep``'s basic pipeline is used on datasets for which there are only t1ws
-and at least one functional (EPI) file, but no SBRefs or fieldmaps.
-To force using this pipeline on datasets that do include fieldmaps and SBRefs
-use the ``--ignore fieldmaps`` flag.
-
-Several steps are added or modified if `Surface preprocessing`_ is enabled.
-
-What It Does
-------------
-High-level view of the basic pipeline:
-
-.. image:: ds005.dot.png
-    :scale: 100%
-
-BIDSDatasource
-~~~~~~~~~~~~~~
-
-This node reads the BIDS_-formatted T1 data.
-
-t1w_preprocessing
-~~~~~~~~~~~~~~~~~
-
-.. image:: t1w_preprocessing.dot.png
-    :scale: 100%
+===========================
+Processing pipeline details
+===========================
+
+``fmriprep`` adapts its pipeline depending on what data and metadata is
+available is used as the input. For example slice time correction will be
```

Comment:
Does this sentence actually say the opposite it meant? I'd suggest (please also check I changed word ordering):

```
For example, slice timing correction will be
performed only if the ``SliceTiming`` metadata field is found for the input dataset
```